packages/api-client/src/api/getCategory/defaultQuery.ts

```diff
@@ -0,0 +1,61 @@
+import gql from 'graphql-tag';
+
+export default gql`
+  query collections {
+	collections{
+      totalItems
+      items {
+        id
+        createdAt
+        updatedAt
+        languageCode
+        name
+        breadcrumbs {
+          id
+          name
+          slug
+        }
+        description
+        position
+        slug
+        featuredAsset {
+          id
+          name
```

Comment:
If you need to actually display the collection image, get the `preview` property.